cmd/root.go

```diff
@@ -106,13 +110,21 @@ func BuildFirewallConfiguration(options *RootOptions) (*iptables.FirewallConfigu
 		return nil, fmt.Errorf("--outgoing-proxy-port must be a valid TCP port number")
 	}
 
+	for _, subnet := range options.SubnetsToIgnore {
+		_, _, err := net.ParseCIDR(subnet)
+		if err != nil {
+			return nil, fmt.Errorf("%s is not a valid CIDR address", subnet)
```

Comment:
ParseCIDR will only return an error message like `invalid CIDR address: 0.0.0.0` (see [this](https://cs.opensource.google/go/go/+/refs/tags/go1.17.3:src/net/ip.go;l=738)), so I decided to roll my own. It's more readable, I hope.

Reply:
I think this looks great! Easy to understand and descriptive 👍🏻